python/pyspark/sql/functions.py

```diff
@@ -1433,8 +1433,12 @@ def timestamp_seconds(col):
     >>> from pyspark.sql.functions import timestamp_seconds
     >>> spark.conf.set("spark.sql.session.timeZone", "America/Los_Angeles")
```

Comment:
do we set the timezone in pyspark doctest, like the scala side test?

Reply:
I'm wondering if this is the common approach we do for all timestamp related pyspark doc tests.

Reply:
Yeah, we can do that too. Let's set it when we face another similar issue. It's not difficult to do it.